modoboa/lib/email_utils.py

```diff
@@ -262,6 +263,47 @@ def render_headers(self, **kwargs):
         return render_to_string("common/mailheaders.html", context)
 
 
+def split_address(address):
+    """Split an e-mail address into local part and domain."""
+    assert isinstance(address, six.text_type),\
+        "address should be of type %s" % six.text_type.__name__
+    if "@" not in address:
+        local_part = address
+        domain = None
+    else:
+        local_part, domain = address.split("@", 1)
```

Comment:
I think we should consider the latest @, not the first. (example : "test@toto.com"@domain.tld) And actually, it was implemented like this before.

Reply:
fixed